### ISSUE TITLE
[bug]: Textual inversion training appears to have no effect

### ISSUE THREAD

**Issue by thirstyhouseplant** [invoke-ai/InvokeAI#2663]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Linux

### GPU

cuda

### VRAM

24GB

### What happened?

I've done seven or so textual inversion runs over the last few days, and it appears that the ti training is having no effect/not learning anything new. This effect has been persistent across different machines, different datasets, and different init and trigger tokens.

Weirder still, it seems as though the initialization token is the only thing that really gets saved. With the default init token `*` and a trigger word of `thirstyhouseplant`, the prompt `A photo of <thirstyhouseplant>` generated many (admittedly nice) images of... stars.

With a default init token of `guy` and a trigger word of `thp`, the prompt `A photo of <thp>` generated many pictures of generic guys. To further test, I ran the prompt again with the same seed and settings, replacing the embedding trigger `<thp>` with literally just `guy`, so `A photo of guy` -- the resulting image was 99.5% identical. 

I loaded the learned_embedding.bin's into an embedding inspector in the A1111 interface. For the TI trained with init token `guy` and trigger `thp`, the following was output:
```
Embedding ID: [1ca5] (loaded)
Step: None
SD checkpoint: None
SD checkpoint name: None
Vector count: 1
Vector size: 768
--------------------------------------------------------------------------------
Vector[0] = tensor([ 0.0093,  0.0051, -0.0037,  ...,  0.0083,  0.0030,  0.0124])
Magnitude: 0.36699968576431274
Min, Max: -0.03741455078125, 0.049896240234375

Similar tokens:
guy</w>(2149)   guy(10008)   guys</w>(1791)   dude</w>(5710)   girl</w>(1611)   person</w>(2533) 
man</w>(786)   guys(43588)   thing</w>(946)   fella</w>(17586)   kid</w>(3551)   bloke</w>(24384)
boy</w>(1876)   people</w>(1047)   lad</w>(10707)   woman</w>(2308)   dudes</w>(14449)
men</w>(1656)   gurl</w>(25445)   guyz</w>(48170)   lady</w>(2909)   bro</w>(4410)
one</w>(637)  mans</w>(7746)   friend</w>(1625)   player</w>(2477)   cop</w>(7070)
dog</w>(1929)   ppl</w>(6758)   girl(3914)
```

Interestingly, for embeddings where `*` was the initializer token (the default setting), the output was:
```
Embedding ID: [4b3b] (loaded)
Step: None
SD checkpoint: None
SD checkpoint name: None
Vector count: 1
Vector size: 768
--------------------------------------------------------------------------------
Vector[0] = tensor([ 0.0145,  0.0031, -0.0079,  ...,  0.0207, -0.0162,  0.0134])
Magnitude: 0.376102089881897
Min, Max: -0.04587707668542862, 0.04200134426355362

Similar tokens:
★</w>(10643)   ★★</w>(36644)   ♥️</w>(3405)   ☆</w>(13439)   ♥️♥️</w>(19604)
♡</w>(6251)   ●</w>(26999)   ★★★★</w>(44034)   ♥️♥️♥️</w>(23255)   \xe2\x96(4168)
⭐</w>(19012)   ♪</w>(12382)   【</w>(12534)   ★★★★★</w>(45984)   |</w>(347)
***</w>(7829)   ✔️</w>(13749)   ★(9339)   ;)</w>(3661)   💥💥💥</w>(48890)
...#</w>(8195)   ♬</w>(24753)   ⭐(6410)   ☆(23941)   +</w>(266)   ♥️♥️(12975)
\xe2\x99(1548)   ☞</w>(31255)   ♫</w>(10814)   ″</w>(25061)
--------------------------------------------------------------------------------
```

The latter output was consistent across the five initial TI runs that all used `*` as an init token. 

Now, one caveat that may have something to do with this is that these were all attempts to train on top of 1.5-based, converted-to-diffusers models. I'm currently doing a run off of vanilla sd-v1.5 to see if that has any impact. 

I had `rsync` running on a loop during all of this work, so I think I may have log files. The GPU was fully utilized throughout the training time (~13 hours or so), so it was clearly doing *something*, but I'm at a loss.

Happy to provide as much additional information as needed. 

![ti-fail](https://user-images.githubusercontent.com/125397583/218836091-a645d81f-be78-4c3f-b970-cd1096fbc5d9.jpeg)

### Contact Details

@thirstyhouseplant on Discord

**Comment by thirstyhouseplant** [comment]
This caught my attention: output in the console when the ti process is first kicked off.
`{'variance_type'} was not found in config. Values will be initialized to default values.` Unsure if it's inconsequential.

And poking around in `hparams.yml`, one thing jumped out to me:
```
accelerator: <accelerate.accelerator.Accelerator object at 0x7f59053d96c0>
adam_beta1: '0.9'
adam_beta2: '0.999'
adam_epsilon: 1e-08
adam_weight_decay: '0.01'
allow_tf32: 'False'
center_crop: 'False'
checkpointing_steps: '500'
enable_xformers_memory_efficient_attention: 'False'
env_local_rank: '-1'
gradient_accumulation_steps: '4'
gradient_checkpointing: 'False'
hub_model_id: None
hub_token: None
initializer_token: guy
initializer_token_id: '2149'
kwargs: '{}' ############################ <<< Should this be empty?
learnable_property: object
learning_rate: '0.0005'
local_rank: '-1'
```
